pgrx-pg-sys/build.rs

```diff
@@ -31,7 +31,7 @@ fn is_nightly() -> bool {
         return false;
     }
     let rustc = env_tracked("RUSTC").map(PathBuf::from).unwrap_or_else(|| "rustc".into());
-    let output = match std::process::Command::new(rustc).arg("--verbose").output() {
+    let output = match std::process::Command::new(rustc).arg("--version").output() {
```

Comment:
🤔

Reply:
that's a hilarious bug.

```
$ rustc --verbose
error: no input filename given

$  rustc --version
rustc 1.69.0 (84c898d65 2023-04-16)
```

Both of those pass the conditional at the end of the function.  🤦